app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,24 @@
+<body>
```

Comment:
Individual view pages don't need a `<body>` tag, since that is included in `application.html.erb`.